src/helics/shared_api_library/backup/helics/helics_api.h

```diff
@@ -143,6 +143,7 @@ typedef enum {
     HELICS_PROPERTY_TIME_OUTPUT_DELAY = 150,
     HELICS_PROPERTY_TIME_STOPTIME = 152,
     HELICS_PROPERTY_TIME_GRANT_TIMEOUT = 161,
+    HELICS_PROPERTY_INT_CURRENT_ITERATION = 258,
```

Comment:
Do you want to include this change to iterations in this PR?

Reply:
looks like the backup file hadn't been updated properly, since this was a change from a previous update